plugins/mongo_db_plugin/mongo_db_plugin.cpp

```diff
@@ -310,10 +310,12 @@ void mongo_db_plugin_impl::queue( Queue& queue, const Entry& e ) {
 }
 
 void mongo_db_plugin_impl::accepted_transaction( const chain::transaction_metadata_ptr& t ) {
+   if( !store_transactions ) {
+      return;
+   }
+
    try {
```

Comment:
Why move these statements out of try/catch? Do you see faster performance?

Reply:
This plugin itself is CPU-bound.  if-return statements are never throw, so it's not necessary in try-block which may have performance costs. (It depends on the compiler)

Reply:
I would be surprised if that made any difference but would be interested if you could point me to anything that demonstrates that.

Reply:
The difference is tiny. It's more a coding style issue. I'll remove the try-catch changes later.